lib/ts/ink_align.h

```diff
@@ -53,6 +53,12 @@ union Alias64 {
 /** Default alignment */
 #define INK_ALIGN_DEFAULT(size) INK_ALIGN(size, INK_MIN_ALIGN)
 
+static inline size_t
+aligned_spacing(size_t len, size_t block = INK_MIN_ALIGN)
+{
+  return INK_ALIGN(len, block);
+}
```

Comment:
This function is superfluous. In every case added here, INK_ALIGN does exactly the same job without asking the reader to comprehend a new function.